src/lib/isRgbColor.js

```diff
@@ -1,12 +1,40 @@
+/* eslint-disable prefer-rest-params */
 import assertString from './util/assertString';
 
 const rgbColor = /^rgb\((([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]),){2}([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\)$/;
 const rgbaColor = /^rgba\((([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]),){3}(0?\.\d|1(\.0)?|0(\.0)?)\)$/;
 const rgbColorPercent = /^rgb\((([0-9]%|[1-9][0-9]%|100%),){2}([0-9]%|[1-9][0-9]%|100%)\)/;
 const rgbaColorPercent = /^rgba\((([0-9]%|[1-9][0-9]%|100%),){3}(0?\.\d|1(\.0)?|0(\.0)?)\)/;
+const startsWithRgb = /^rgba?/;
 
-export default function isRgbColor(str, includePercentValues = true) {
+export default function isRgbColor(str, options) {
   assertString(str);
+  // remove spaces
+  let strict, includePercentValues;
+
+  if (typeof options !== 'object') {
+    // backward compaitable behaviour
+    strict = true;
+    includePercentValues = true;
+    if (arguments.length >= 2) {
+      includePercentValues = arguments[1];
+    }
+    if (arguments.length >= 3) {
+      strict = arguments[2];
+    }
+  } else {
```

Comment:
You're not reading the actual options here. They are just being set to true. That defeats the purpose of having the options object in the first place. Having them default to true is not the same as hard coding them to always be true.

Reply:
@braaar I am not sure what you mean, as demonstrated by the tests the values are being read from options object, i have added more cases in [c890403](https://github.com/validatorjs/validator.js/pull/2029/commits/c89040398f668fc73fe2d39164a9749962ce0a33)

Reply:
Sorry, I had it mixed up a little. I'm making some proper suggestions now.